snap/hooks/configure

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh -e
```

Comment:
Is using DASH here due to missing bash?
Most of our shell scripts are in Bash... it would be great to keep one shell in use... as syntax is slightly different. Sure if bash is missing here, fine for sh/dash.